energy_profiles_explorer.py

```diff
@@ -0,0 +1,108 @@
+"""
+This script sets up a Streamlit app 'Energy-use profiles explorer'.
+
+The main function is energy_profiles_explorer() which initializes the app, sets the page configuration, and creates a sidebar menu for navigation. The sidebar allows users to navigate between different pages:
+- "About the explorer"
+- "Explore the results"
+- "Explore an energy-use profile"
+- "About the data"
+
+To run this app localy:
+streamlit run energy_profiles_explorer.py
+"""
+
+## Package imports
+import streamlit as st  # For building the web app
+import pandas as pd  # For simple data manipulation
+from PIL import Image  # For loading images
+import altair as alt  # For creating interactive charts with Altair
+from streamlit_option_menu import option_menu  # For creating a sidebar menu
+import os
+
+# Local imports
+from config.fonts_setup import nestafont, NESTA_COLOURS
+from config.css_style import set_css_style
+from pages.about_dashboard_page import about_dashboard_page
+from pages.overview_results_page import overview_results_page
+from pages.select_profile_page import select_profile_page
+from pages.about_data_page import about_data_page
+
+# Setting up themes and fonts
+alt.themes.register("nestafont", nestafont)
+alt.themes.enable("nestafont")
+set_css_style()
+
+# Get the current directory to load images and other resources
+current_dir = os.getcwd()
+
+
+def set_up_sidebar(nesta_logo):
```

Comment:
```suggestion
def set_up_sidebar():
```

Perhaps this was for a previous version, but this argument isn't used in this function.